notes.md

```diff
@@ -0,0 +1,115 @@
+## Planning for Bowling Challenge
+
+Outcome of whole project is to have a program that can take scores for a whole 10 frames of bowling.
+- It can take an input score from the user for each bowl of each frame
+- It can determine if that score constitutes a strike or spare
+- It cans store the total score and return it to the user
+- It can determine if any bonuses are due for strikes/spares and return add them to the total score
+- It can determine the game conditions of the 10th frame and give extra bowls if strikes or spares are scored
+- It can determine the game is over and inform the user providing their final score
+- Edge cases:
+* Scores given are negatives/above 10
+
+User Stories
+
+As a player
+So I can enjoy a game of bowling
+I want to be able to enter my score
+
+As a player
+So I can enjoy a game of bowling
+I want my score to be returned to me after each frame
+
+As a player
+So I can know my final score
+I want my score to be saved
+
+As a player
+So I can have my correct score
+I want to be given bonuses for bowling strikes and spares
+
+As a player
+So I can enjoy the full game
+I want to be able to have an extra bowl in the 10th frame if applicable
+
+As a player
+So I can enjoy a game of bowling
+I want to know when the game is over
```

Comment:
Nice idea breaking down the problem into user stories. It's clear to see what features you want to implement